docs/rationale.md

```diff
@@ -143,6 +143,14 @@ One final thing: TC39 has [not yet decided if decorators come before or after `e
 export @decorator class Foo {}
 ```
 
+### Template literals
```

Comment:
Can you add code example to here same as changelog?

Reply:
Done.